### PR TITLE
Quickstart intro cleanup

### PR DESCRIPTION
**Pull request by yvonneanne**
Following advice from #docs-working-group

### CODE REVIEW

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 1]
```diff
@@ -7,34 +7,34 @@
...
+The _Quick Start_ tutorial assumes that you are installing the {sdk-short-name} for the first time and illustrates how to create, build, and deploy your first dapp. 
 
 To keep the instructions simple and focused on the task at hand, the _Quick start_ is split into two scenarios:
 
-* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample application with {IC} processes that run _locally to simulate connecting to the network_.
+* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample dapp with processes that run _locally to simulate connecting to the Internet Computer_.
 
-* In the link:network-quickstart{outfilesuffix}[Network deployment] scenario, you create the sample application locally but _connect to the {IC} running remotely_ on computers in an external data center.
+* In the link:network-quickstart{outfilesuffix}[IC deployment] scenario, you create the sample dapp locally but _connect to the {IC} blockchain_ running node machines in independent data centers and deploy the dapp there.
```
Comment (yvonneanne) [inline modules/quickstart/pages/quickstart-intro.adoc:17]:
```suggestion
* In the link:network-quickstart{outfilesuffix}[On-chain deployment] scenario, you create the sample dapp locally but _connect to the {IC} blockchain_ running node machines in independent data centers and deploy the dapp there.
```

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 2]
```diff
@@ -7,34 +7,34 @@
...
 
-The default application consists of back-end code written in  {proglang}, a programming language specifically designed for interacting with the {IC}, and a simple webpack-based front-end.
+The default dapp consists of back-end code written in  {proglang}, a programming language specifically designed for interacting with the {IC}, and a simple webpack-based front-end.
 
 == Getting ICP tokens and cycles
 
-As discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], *cycles* are required to power canister operations for applications running on the {IC}. 
-As a developer, you have a few different options for acquiring and managing cycles for your applications:
+As discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], *cycles* are required to power canister smart contract operations for dapps running on the {IC}.
```
Comment (yvonneanne) [inline modules/quickstart/pages/quickstart-intro.adoc:30]:
```suggestion
As discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], *cycles* are required to power canister smart contract operations for dapps running on the {IC} blockchain. 
```

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 3]
```diff
@@ -7,34 +7,34 @@
 ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 
 [[quick-start-intro]]
-This {sdk-long-name} provides tools, sample code, and documentation to help you create programs to run on the {IC}.
-The _Quick Start_ tutorial assumes that you are installing the {sdk-short-name} for the first time and illustrates how to create, build, and deploy your first application. 
+This {sdk-long-name} provides tools, sample code, and documentation to help you create canister smart contract dapps to run on the {IC}.
```
Comment (andreacerulli) [inline modules/quickstart/pages/quickstart-intro.adoc:10]: Maybe "canister smart contracts" without "dapps" is sufficient.

```suggestion
This {sdk-long-name} provides tools, sample code, and documentation to help you create canister smart contract dapps and deploy them on the {IC} blockchain mainnet.
```

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 4]
```diff
@@ -7,34 +7,34 @@
...
+The _Quick Start_ tutorial assumes that you are installing the {sdk-short-name} for the first time and illustrates how to create, build, and deploy your first dapp. 
 
 To keep the instructions simple and focused on the task at hand, the _Quick start_ is split into two scenarios:
 
-* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample application with {IC} processes that run _locally to simulate connecting to the network_.
+* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample dapp with processes that in a _local canister execution environment_.
 
-* In the link:network-quickstart{outfilesuffix}[Network deployment] scenario, you create the sample application locally but _connect to the {IC} running remotely_ on computers in an external data center.
+* In the link:network-quickstart{outfilesuffix}[On-chain deployment] scenario, you create the sample dapp locally but _connect to the {IC} blockchain_ running on node machines in independent data centers and deploy the dapp there.
```
Comment (andreacerulli) [inline modules/quickstart/pages/quickstart-intro.adoc:17]: Based on [this](https://dfinity.slack.com/archives/C02FV87QYKS/p1632389444118200), I started using "{IC} blockchain mainnet" when is about deploying canisters.

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 5]
```diff
@@ -7,34 +7,34 @@
...
+As a developer, you have a few different options for acquiring and managing cycles for your dapps:
 
 * Purchase or claim ICP tokens through an exchange that lists ICP tokens available for trade, then convert your tokens to cycles using the link:network-quickstart{outfilesuffix}#convert-tokens[{sdk-short-name} command-line interface] or the link:https://nns.ic0.app/#/auth[Network Nervous System application].
 * Register for cycles by signing up for an account or link:../developers-guide/default-wallet{outfilesuffix}#wallet-create-wallets[cycles wallet] through a provider offering those services.
-* Coordinate with other developers to link:../developers-guide/default-wallet{outfilesuffix}#wallet-send[send] and link:../developers-guide/default-wallet{outfilesuffix}#wallet-receive[receive] cycles directly to and from canisters through a cycles wallet or another application.
+* Coordinate with other developers to link:../developers-guide/default-wallet{outfilesuffix}#wallet-send[send] and link:../developers-guide/default-wallet{outfilesuffix}#wallet-receive[receive] cycles directly to and from canisters through a cycles wallet or another dapps.
```
Comment (andreacerulli) [inline modules/quickstart/pages/quickstart-intro.adoc:35]:
```suggestion
* Coordinate with other developers to link:../developers-guide/default-wallet{outfilesuffix}#wallet-send[send] and link:../developers-guide/default-wallet{outfilesuffix}#wallet-receive[receive] cycles directly to and from canisters through a cycles wallet or another canister.
```

**Review by andreacerulli — APPROVED** [review]
LGTM

File: modules/quickstart/pages/quickstart-intro.adoc (hunk 4 above)
Comment (yvonneanne) [reply modules/quickstart/pages/quickstart-intro.adoc:17]: Thanks! I'll use it with "mainnet" once and then {platform}, so it should be easy to replace if we ever change our marketing strategy again :)

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 6]
```diff
@@ -7,34 +7,34 @@
...
+The _Quick Start_ tutorial assumes that you are installing the {sdk-short-name} for the first time and illustrates how to create, build, and deploy your first dapp. 
 
 To keep the instructions simple and focused on the task at hand, the _Quick start_ is split into two scenarios:
 
-* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample application with {IC} processes that run _locally to simulate connecting to the network_.
+* In the link:local-quickstart{outfilesuffix}[Local development] scenario, you create and deploy the sample dapp with processes that in a _local canister execution environment_.
 
-* In the link:network-quickstart{outfilesuffix}[Network deployment] scenario, you create the sample application locally but _connect to the {IC} running remotely_ on computers in an external data center.
+* In the link:network-quickstart{outfilesuffix}[On-chain deployment] scenario, you create the sample dapp locally but _connect to the {IC} blockchain_ running on node machines in independent data centers and deploy the dapp there.
```
Comment (yvonneanne) [inline modules/quickstart/pages/quickstart-intro.adoc:17]:
```suggestion
* In the link:network-quickstart{outfilesuffix}[On-chain deployment] scenario, you create the sample dapp locally but _connect to the {platform}_ running on node machines in independent data centers and deploy the dapp there.
```

File: modules/quickstart/pages/quickstart-intro.adoc [hunk 7]
```diff
@@ -7,34 +7,34 @@
...
 
-The default application consists of back-end code written in  {proglang}, a programming language specifically designed for interacting with the {IC}, and a simple webpack-based front-end.
+The default dapp consists of back-end code written in  {proglang}, a programming language specifically designed for interacting with the {IC}, and a simple webpack-based front-end.
 
 == Getting ICP tokens and cycles
 
-As discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], *cycles* are required to power canister operations for applications running on the {IC}. 
-As a developer, you have a few different options for acquiring and managing cycles for your applications:
+As discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], *cycles* are required to power canister smart contract operations for dapps running on the {IC} blockchain.
```
Comment (yvonneanne) [inline modules/quickstart/pages/quickstart-intro.adoc:30]:
```suggestion
As discussed in link:../developers-guide/concepts/tokens-cycles{outfilesuffix}[Tokens and cycles], *cycles* are required to power canister smart contract operations for dapps running on the {platform}. 
```

**Pull request merged**